Wikipedia/Code/WMFTableOfContentsViewController.swift

```diff
@@ -224,8 +224,8 @@ open class WMFTableOfContentsViewController: UIViewController, UITableViewDelega
         tableOfContentsFunnel.logOpen()
     }
     
-    open override func viewDidDisappear(_ animated: Bool) {
-        super.viewDidDisappear(animated)
+    open override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
         deselectAllRows()
```

Comment:
Is it necessary to deselect all rows at all? If I background the app and come back, I'm scrolled to the right section but the selected section in the TOC is off

Reply:
hmm.. wonder if my tweak to [TOC fragment logic](https://github.com/wikimedia/wikipedia-ios/pull/3230) affects this? will check

Reply:
i may know a fix for the TOC issue @nambatee noticed... checking...

Reply:
Sounds good, this was just to get rid of that console error I was seeing, didn't affect the editor presentation dismissal upon backgrounding.

Reply:
TOC selection reproducible on 6.3. Filing separate ticket with repro steps.